runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/BrowserTargetWidget/BrowserTargetWidget.tsx

```diff
@@ -52,11 +52,14 @@ const BrowserTargetWidget: FC<WidgetProps> = (props) => {
             const prefix = Object.keys(PREFIX_OPTIONS).find((option) => props.value.startsWith(option)) || 'None';
             const value = prefix === 'None' ? props.value : props.value.replace(prefix, ''); 
             setFormState({ prefix: prefix, value: value});
+            onChangeWrapper(prefix, value);
         }
     }, []);
 
     const onChangeWrapper = (prefix: string, value: string) => {
-        if(prefix === 'None') {
+        if (!prefix || !value) {
+            props.onChange(null);
+        } else if (prefix === 'None') {
             props.onChange(value);
         } else{
             props.onChange(prefix + value);
```

Comment:
what if value is "sth" and prefix is false/null? this way -> propsOnChange(null)

Reply:
![LeonardoDicaprioSoldGifGIF](https://github.com/runbotics/runbotics/assets/69820069/2629360d-5e4d-4998-a318-132f18dce65a)